sormas-ui/src/main/java/de/symeda/sormas/ui/utils/components/page/title/TitleLayoutHelper.java

```diff
@@ -12,7 +12,11 @@ private TitleLayoutHelper() {
 
 	public static StringBuilder buildPersonString(PersonDto person) {
 		StringBuilder personString = new StringBuilder();
-		String personFullName = PersonDto.buildCaption(person.getFirstName(), person.getLastName());
+		String personFullName = "";
+		if (!person.isPseudonymized()) {
```

Comment:
if the person is pseudonymized then the fillname should be "Confidential" I would say

Reply:
updated